MonoGame.Framework/Directory.Build.props

```diff
@@ -1,8 +1,15 @@
 <Project>
   <PropertyGroup>
-    <MonoGamePlatform>$(MSBuildProjectName.Substring(9))</MonoGamePlatform>
```

Comment:
Neat trick! Won't work for Net projects though. (but we should drop those anyway)

Reply:
You added that XD

Reply:
😅